dz12/src/app.js

```diff
@@ -0,0 +1,95 @@
+function checkIfTheNumberIsEvenOrOdd(number) {
+  try {
+    if (!Number.isInteger(number)) {
+      throw new Error(`${number} - не является целым числом`)
+    }
+    if (number % 2 === 0) {
+      return 'Четное'
+    }
+    return 'Нечетное'
+  } catch (error) {
+    return `${number} - не является целым числом`
+  }
+}
+
+function searchForCarWithGivenColor(cars, color) {
+  const car = cars.find((findCar) => findCar.color === color)
+  if (car) {
+    return true
+  }
+  return false
```

Comment:
```suggestion
  return Boolean(car)
```

можно чуть сократить и явно привести к Boolean. либо воспользоваться `some` (будет в следующем уроке), который сразу вернёт true/false